hilti/toolchain/src/base/util.cc

```diff
@@ -19,6 +19,14 @@
 
 // We include pathfind directly here so we do not have to work
 // around it being installed by its default install target.
+
+#ifdef __linux__
+#ifndef __gnu_linux__
+// pathfind checks for __gnu_linux__, which isn't set on Alpine.
```

Comment:
Let's link https://github.com/bkloppenborg/pathfind/issues/5 here.

Reply:
Upstream is fixed, so updating the submodule instead.